_plugins/git_info.rb

```diff
@@ -0,0 +1,4 @@
+# Make version information from git available via site.git_info in templates
+Jekyll::Hooks.register :site, :pre_render do |site, payload|
+    site.config['git_info'] = `git describe --tags`
+end
```

Comment:
missing ending newlines in the few files.

Reply:
rebased against main where I added them in response to the other PR; they should (hopefully) be there now